app/models/comment.rb

```diff
@@ -1,4 +1,11 @@
 class Comment < ApplicationRecord
   belongs_to :author, class_name: 'User', foreign_key: 'user_id'
```

Comment:
- [ ] This issue here has not been addressed, your comments table (in the schema) knows a user as a user, with an ID of `user_id` and not `author_id`. The only table to carry the name `author_id` as the foreign key is the post table. You can change the `belongs_to :author` line to `belongs_to :user`.

Reply:
![comment](https://github.com/danielochuba/duo-blog/assets/81446846/d2d8ced8-df9d-4e3c-9141-4c9f0fbb2df9)

@elisha2kyakpo1 what do you have to say about this

I invited you on a zoom call for further explanations